src/applications/gi/actions/index.js

```diff
@@ -180,11 +184,34 @@ export function fetchProfile(facilityCode, version) {
     dispatch({ type: FETCH_PROFILE_STARTED });
 
     return fetch(url, api.settings)
-      .then(res => res.json())
-      .then(
-        payload => withPreview(dispatch, { type: FETCH_PROFILE_SUCCEEDED, payload }),
-        err => dispatch({ type: FETCH_PROFILE_FAILED, err })
-      );
+      .then(res => {
+        if (res.ok) {
+          return res.json();
+        }
+
+        return res.json()
+          .then((errors) => {
+            throw new Error(errors);
+          });
+      })
+      .then(payload => {
+        const institutionZIP = _.get(payload, 'data.attributes.zip');
+        const bahUrl = `${api.url}/zipcode_rates/${institutionZIP}`;
+
+        return fetch(bahUrl, api.settings)
```

Comment:
- Grandfathered rate is a new concept returned by the zip code end point
- calculation currently uses the non-grandfathered rate from the institution end point 
- institution end point was not updated to have the grandfathered data (apparently the rate on the institution end point comes from a different data source than the rate from used on the zip end point)
- this messiness requires calling the zip end point to get the grandfathered rate for the institution zip

Decided the simplest approach would be to chain a call to the zip end point off of the result of the institution call. A better experience would have been to call this when the user clicked the relevant question but I wanted to avoid spreading this complexity around the app.